app/kumactl/cmd/apply/apply_test.go

```diff
@@ -372,15 +375,20 @@ var _ = Describe("kumactl apply", func() {
 		Expect(resource.Spec.Networking.Address).To(Equal("1.1.1.1"))
 
 		// then
+		// we need to marshall time 0 it manually since it seems to produce different results across operating system
+		zeroTime, err := yaml.Marshal(time.Unix(0, 0))
```

Comment:
I don't really understand the difference. Why we can use hardcoded date like `"2018-07-17T16:05:36.995+00:00"` here?

Reply:
Because of this
```
func (m meta) GetCreationTime() time.Time {
	return time.Unix(0, 0) // the date doesn't matter since it is set on server side anyways
}

func (m meta) GetModificationTime() time.Time {
	return time.Unix(0, 0) // the date doesn't matter since it is set on server side anyways
}
```
in `apply.go` file

Reply:
If `time.Unix(0,0)` is OS specific, maybe we can return `time.Parse(...)` with some constant in these methods?

Reply:
it was a timezone. Changing to UTC fixed the problem